### PR TITLE
Verify if gasLimit is enough to execute message

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-contracts/contracts/wallet/Executor.sol [hunk 1]
```diff
@@ -48,6 +48,7 @@ contract Executor {
         bytes memory signatures) public returns (bytes32)
     {
         uint256 startingGas = gasleft();
+        require(gasLimitExecution <= startingGas, "Too low gas limit");
```
Comment (marekkirejczyk) [inline universal-login-contracts/contracts/wallet/Executor.sol]: Gas limit too low

Comment (JustynaBroniszewska) [reply universal-login-contracts/contracts/wallet/Executor.sol]: ✅

File: universal-login-contracts/contracts/wallet/Executor.sol [hunk 2]
```diff
@@ -48,6 +48,7 @@ contract Executor {
         bytes memory signatures) public returns (bytes32)
     {
         uint256 startingGas = gasleft();
+        require(gasLimitExecution <= startingGas, "Gas limit too low");
```
Comment (marekkirejczyk) [inline universal-login-contracts/contracts/wallet/Executor.sol]: "Relayer set gasLimit too low"

File: universal-login-contracts/test/contracts/Executor/Executor-refund.ts [hunk 3]
```diff
@@ -58,15 +57,15 @@ describe('CONTRACT: Executor - refund', async () => {
         nonce: 0,
         gasPrice: 1,
         gasToken: '0x0',
-        gasLimit: TEST_PAYMENT_OPTIONS.gasLimit
+        gasLimit: utils.bigNumberify('120000')
```
Comment (marekkirejczyk) [inline universal-login-contracts/test/contracts/Executor/Executor-refund.ts]: extract constant?

Comment (marekkirejczyk) [reply universal-login-contracts/test/contracts/Executor/Executor-refund.ts]: what is the difference between 80k and 120k?

File: universal-login-relayer/lib/core/utils/messages/serialisation.ts [hunk 4]
```diff
@@ -7,6 +7,8 @@ import MessageItem from '../../models/messages/MessageItem';
 
 const {executeSigned} = new utils.Interface(WalletContract.interface).functions;
 
+export const GAS_LIMIT_MARGIN = 40000;
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/core/utils/messages/serialisation.ts]: GAS_LIMIT_BASE -> Move to separate file

**Pull request merged**